docs/ibexa_engage/ibexa_engage.md

```diff
@@ -0,0 +1,85 @@
+
+![Welcome page - form](img/welcome_page_form.png "Welcome page - form")
+
+[[= product_name_base =]] team creates and provides user account. An invitation link is sent during the setup process.
+Once the registration and configuration process is completed, the welcome page displays information about campaign tools available with [[= product_name_engage =]] and link to [Qualifio documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Campaigns) where you can get information about creating and managing campaigns.
+
+![Welcome page - active account](img/welcome_page_activated.png "Welcome page - active account")
+
+Click **Manage campaigns in Qualifio** button to open Qualifio page, where you can create and manage campaigns.
+
+![Qualifio Manager](img/qualifio_manager.png "Qualifio Manager - campaign list")
+
+## Campaign
+
+A campaign is a set of different concepts, grouped per steps, and open to configuration.
+You can use campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
+
+Campaign can contain a welcome screen, a form step, an interaction element, an exit screen, and so on.
+
+You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns.
+Users can configure the backgrounds, themes, or designs, and set up a specific time frame for each campaign.
+
+Each campaign comes with a unique ID - string of numbers, that the platform automatically defines during the creation of campaign. Campaign ID is a reference to the campaign in the applications, services and data connections.
+
+There are two types of campaigns:
+
+- **Anonymous** - engages with the audience, increases the advertising impressions, or collects cookies of personalized audiences.
+- **Identified** - gathers information on the user by filling out an identification form.
+
+## Publication channel
+
+Every campaign includes a minimum of one publication channel that you can choose from the three options the platform provides for publishing a campaign:
+
+- **Widget/iframe** - uses a JavaScript code or an HTML iframe from Qualifio. Additionally, you can use oEmbed, Facebook Instant articles, and AMP technologies to publish iframes. You need to manually paste the code into your website or CMS.
+
+- **Minisite** - uses a unique URL from Qualifio that can hosts the campaign. This URL points to your subdomain.
+
+- **Mobile** - uses a link from Qualifio to integrate a campaign that is optimized for mobile usage.
+
+!!! note
+    By default, every type of publication channel is responsive and compatible with all mobile devices.
+
+## Work with campaigns
+
+You can create, configure, and create campaigns using [Qualifio Engage](https://developers.qualifio.com/docs/engage/).
+To access the Qualifio Back Office page and create or manage campaigns, click the **Manage campaigns in Qualifio** button under the **Ibexa Engage** menu.
```

Comment:
Is it "Qualifio Back Office", or could we use the ""Qualifio Manager" term that you introduced above?
Also, I'd try to describe navigation to the button in the following manner: "go to the Back Office, **Ibexa Engage**, and click the **Manage campaigns in Qualifio** button.